src/components/organisms/EarthEditor/PropertyPane/hooks.ts

```diff
@@ -39,21 +39,14 @@ export type FieldPointer = {
 };
 
 export default (mode: Mode) => {
-  const [selected] = useSelected();
+  const [selected, select] = useSelected();
```

Comment:
```suggestion
  const [selected] = useSelected();
```
I don't think the select is being used.